documentation/modules/ref-broker-component-sizing.adoc

```diff
@@ -0,0 +1,114 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-configuration-sizing-guide.adoc
+//
+[id='broker-component-sizing-{context}']
+= Broker component sizing
+
+Brokers are configured in the `BrokeredInfraConfig` and `StandardInfraConfig` resources. When sizing a broker, the following must be taken into consideration:
+
+* The average message size
+* The number of messages stored
+* The number of queues and/or topics
+* The address full policy
+
+NOTE: In {ProductName} it is only possible to restrict the total amount of memory allocated for a broker, and not restrict memory used by individual addresses.
```

Comment:
Aside: we should experiment with <max-size-bytes-reject-threshold/> of BLOCK mode and consider exposing it to the EnMasse user (see ARTEMIS-2505, where the wiring of this got fixed)